includes/class-amp-story-post-type.php

```diff
@@ -206,6 +206,8 @@ public static function register() {
 
 		add_filter( 'wp_kses_allowed_html', array( __CLASS__, 'filter_kses_allowed_html' ), 10, 2 );
 
+		add_filter( 'safe_style_css', array( __CLASS__, 'filter_safe_style_css' ), 10, 1 );
```

Comment:
Can this be limited to the story post type?

Reply:
We could check the `rest_route` from `$_REQUEST` to detect if it's for the story post type.

Reply:
Added the check within c1755bf.